src/main/java/org/zalando/nakadi/domain/EventTypeSchema.java

```diff
@@ -37,6 +46,24 @@ public void setSchema(final String schema) {
         this.schema = schema;
     }
 
+    @Nullable
```

Comment:
If this property being a null ia a bug situation, then why is it marked as `@Nullable` ?